bitbots_hcm/src/bitbots_hcm/fall_checker.py

```diff
@@ -94,3 +94,29 @@ def predict(self, x):
             prediction = self.check_falling(entry[3:6], entry[6:9])
             y.append(prediction)
         return y
+
+    def check_fallen(self, smooth_accel, not_much_smoothed_gyro):
+        """Check if the robot has fallen and is lying on the floor. Returns animation to play, if necessary."""
+        # Checks if the robot is still moving.
+        if any(abs(n) >= 0.1 for n in not_much_smoothed_gyro):
+            return None
+
+        # Decides which side is facing downwards.
+        if smooth_accel[0] < -7:
```

Comment:
Why is this value not configurable?

Reply:
because it always worked perfect and we never needed to change it